### PR TITLE
Contentlength always null #8

### PR DESCRIPTION
**Pull request by luytk12**
Cannot check on contentlength because it is always null

### CODE REVIEW

**Comment by luytk12** [comment]
this commit fixed issue #8

File: src/NCDO/CDO.cs [hunk 1]
```diff
@@ -635,8 +635,7 @@ public void Sort()
             request.Success = response.IsSuccessStatusCode;
             request.ResponseMessage = response;
             //https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding
-            if ((response.Headers.TransferEncodingChunked.HasValue && response.Headers.TransferEncodingChunked.Value) ||
-                response.Content.Headers.ContentLength > 0)
+            if (response.IsSuccessStatusCode)
```
Comment (RobinHerbots) [inline src/NCDO/CDO.cs:638]: This will not garantee that there is data in the response.

**Review by RobinHerbots — COMMENTED** [review]
See  in code comment.

File: src/NCDO/CDO.cs (hunk 1 above)
Comment (luytk12) [reply src/NCDO/CDO.cs:638]: why check for data in the response ?

**Pull request closed without merging**